pkg/cli/image/manifest/manifest.go

```diff
@@ -284,7 +284,7 @@ func FirstManifest(ctx context.Context, from imagereference.DockerImageReference
 		return nil, ManifestLocation{}, err
 	}
 	if len(srcManifests) == 0 {
-		return nil, ManifestLocation{}, fmt.Errorf("filtered all images from manifest list")
+		return nil, ManifestLocation{}, fmt.Errorf("filtered all images from manifest list and did not retrieve a valid source")
```

Comment:
`manifest list does not have images that match filtering criteria`?

Reply:
updated the msg